src/common.css

```diff
@@ -0,0 +1,413 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+*{
+    font-family: 'Nunito', sans-serif;
+    margin: 0px 0px; 
+    padding: 0px 0px;
+    box-sizing: border-box;
+}
+
+/* IGNORE */
+.cards{
+    display: flex;
+    flex-wrap: wrap;
+    margin: 50px;
+}
+
+.card-container{
+    display: grid;
+    gap: 20px;
+    margin: 50px 100px;
+    height: fit-content;
+    /* width: fit-content; */
+}
+
+.card-container h1:first-child{
+    font-size: 25px;
+    border-bottom: 1px solid var(--secondary-gray);
+}
+
+:root{
+    /* COLORS */
+    --primary-yellow: hsl(35, 90%, 63%);
+    --primary-red: hsl(350, 71%, 43%);
+    --primary-green: hwb(153 27% 59%);
+    --primary-blue: hsl(215, 65%, 38%);
+    --primary-gray: hsl(0, 0%, 20%);
+    
+
+    --secondary-yellow: hsl(35, 89%, 86%);
+    --secondary-red: hsl(349, 55%, 80%);
+    --secondary-green: hsl(150, 10%, 65%);
+    --secondary-blue: hsl(214, 85%, 81%);
+    --secondary-gray: hsl(0, 0%, 70%);
+
+    --tertiary-yellow: hsl(37, 80%, 98%);
+    --tertiary-red: hsl(351, 47%, 97%);
+    --tertiary-green: hsl(156, 10%, 90%);
+    --tertiary-blue: hsl(206, 80%, 96%);
+    --tertiary-gray: hsl(0, 0%, 95%);
+
+    --box-shadow: 0px 0px 25px hsla(0, 0%, 0%, 0.1);
+    /* --box-shadow-tab: 3px 0px 4px hsla(0, 0%, 0%, 0.05); */
+
+    --transition: 300ms ease-in-out;
+}
+
+/* GENERAL */
+h1{
+    font-weight: 700;
+    font-size: 20px;
+}
+
+h2{
+    font-weight: 700;
+    font-size: 18px;
+    color: var(--primary-blue);
+}
+
+h3{
+    font-weight: 700;
+    font-size: 15px;
+}
+
+h4{
+    font-weight: 700;
+    font-size: 12px;
+    font-size: 15px;
+}
+
+p{
+    font-family: 'Lato', sans-serif;
+    font-weight: 300;
+    font-size: 12px;
+    font-size: 15px;
+}
+
+li{
+    text-decoration: none;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+a{
+    text-decoration: none;
+    font-size: 14px;
+}
+
+a:hover{
+    text-decoration: underline;
+}
+
+/* MARGIN-0 */
+.margin-0-auto{
+    margin: 0 auto;
+}
+
+/* BUTTONS */
+.btn{
+    border: none;
+    text-align: center;
+    text-decoration: none;
+    padding: 5px 15px;
+    background-color: var(--primary-blue);
+    color: #fff;
+    width: fit-content;
+    height: min-content;
+    font-weight: 700;
+    font-size: 14px;
+    border-radius: 10px;
+}
+
+.btn:hover{
+    cursor: pointer;
+}
+
+.btn-send{
+    border-radius: 10px;
+}
+
+.btn-reply{
+    border-radius: 10px 0 0 10px;
+}
+
+.btn-react{
+    border-radius: 0 10px 10px 0;
+}
+.btn-close{
+    background: none;
+    color: var(--primary-gray);
+    padding: 0;
+}
+
+/* USER ICONS */
+.user-icon{
+    /* USER ICON SIZE */
+    --size: 25px;
+
+    width: var(--size);
+    height: var(--size);
+    line-height: var(--size);
+
+    background-color: var(--tertiary-gray);
+    
+    /* display: flex; */
+    border-radius: 10000px;
+    text-align: center;
+}
+
+.user-icon p {
+    font-size: calc(var(--size)/2.5);
```

Comment:
Wouldn't it be more readable to create a scale for font-sizes (introduce variables) instead of using the calc() function?

Reply:
we did this common.css line 147 to 167 we added a variable saying that the font-size should be relative to the size of the icon and then scaled it with em in the child element, such that 0.4em sets the font to fit in the user icon frame.